package com.test.moviehub.domain.exceptions;

import java.lang.System;

@kotlin.Metadata(mv = {1, 4, 2}, bv = {1, 0, 3}, k = 2, d1 = {"\u0000\n\n\u0000\n\u0002\u0010\u000e\n\u0002\b\b\"\u000e\u0010\u0000\u001a\u00020\u0001X\u0082T\u00a2\u0006\u0002\n\u0000\"\u000e\u0010\u0002\u001a\u00020\u0001X\u0082T\u00a2\u0006\u0002\n\u0000\"\u000e\u0010\u0003\u001a\u00020\u0001X\u0082T\u00a2\u0006\u0002\n\u0000\"\u000e\u0010\u0004\u001a\u00020\u0001X\u0082T\u00a2\u0006\u0002\n\u0000\"\u000e\u0010\u0005\u001a\u00020\u0001X\u0082T\u00a2\u0006\u0002\n\u0000\"\u000e\u0010\u0006\u001a\u00020\u0001X\u0082T\u00a2\u0006\u0002\n\u0000\"\u000e\u0010\u0007\u001a\u00020\u0001X\u0082T\u00a2\u0006\u0002\n\u0000\"\u000e\u0010\b\u001a\u00020\u0001X\u0082T\u00a2\u0006\u0002\n\u0000\u00a8\u0006\t"}, d2 = {"BAD_RESPONSE_ERROR_MESSAGE", "", "CANCELED_BY_USER", "EMPTY_RESPONSE_ERROR_MESSAGE", "NOT_DEFINED_ERROR_MESSAGE", "NO_CONNECTION_ERROR_MESSAGE", "TIME_OUT_ERROR_MESSAGE", "TOO_MANY_ATTEMPTS", "UNAUTHORIZED_ERROR_MESSAGE", "app_debug"})
public final class ErrorModelKt {
    private static final java.lang.String NO_CONNECTION_ERROR_MESSAGE = "NO_CONNECTION_ERROR_MESSAGE";
    private static final java.lang.String BAD_RESPONSE_ERROR_MESSAGE = "BAD_RESPONSE_ERROR_MESSAGE";
    private static final java.lang.String TIME_OUT_ERROR_MESSAGE = "TIME_OUT_ERROR_MESSAGE";
    private static final java.lang.String EMPTY_RESPONSE_ERROR_MESSAGE = "EMPTY_RESPONSE_ERROR_MESSAGE";
    private static final java.lang.String NOT_DEFINED_ERROR_MESSAGE = "NOT_DEFINED_ERROR_MESSAGE";
    private static final java.lang.String UNAUTHORIZED_ERROR_MESSAGE = "UNAUTHORIZED_ERROR_MESSAGE";
    private static final java.lang.String CANCELED_BY_USER = "CANCELED_BY_USER";
    private static final java.lang.String TOO_MANY_ATTEMPTS = "TOO_MANY_ATTEMPTS";
}